src/features/query-bar/components/filter/FilterButton.js

```diff
@@ -148,15 +161,23 @@ class FilterButton extends React.Component<Props, State> {
         }
     };
 
+    updateSelectedConnector = (option: SelectOptionType) => {
+        const connector = option.displayText;
+
+        this.setState({
+            selectedConnector: connector,
```

Comment:
This should be set to option.key in case displayText changes in the future.

Unlike the displayText, the key absolutely should not change, ever.

Reply:
I'll be using option.value instead of option.key for now since that's how the flowtype is defined for a SelectOptionType